roles/promtail/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+promtail_config_file: /etc/promtail-config.yml
+promtail_version: "1.5.0"
+promtail_install_dir: "/usr/local/bin/promtail"
+promtail_exec_binary: "{{ loki_server_install_dir }}/promtail"
+promtail_binary_url: "https://github.com/grafana/loki/releases/download/v{{ promtail_version }}/promtail-linux-amd64.zip"
+loki_server_url: "http://127.0.0.1:3100/loki/api/v1/push"
+promtail_listen_port: "9080"
+postgresql_log_files_path: "/var/log/test/*.log"
+nginx_log_files_path: "/var/log/test/*.log"
+elasticsearch_log_files_path: "/var/log/test/*.log"
+rabbitmq_log_files_path: "/var/log/test/*.log"
+redis_log_files_path: "/var/log/test/*.log"
+couchdb_log_files_path: "/var/log/test/*.log"
+couchdb2_proxy_log_files_path: "/var/log/test/*.log"
+commcare_log_files_path: "var/log/test/*.log"
```

Comment:
I think we'd want to exclude certain logs to avoid getting duplicates. (or else we could look at changing our commcare log settings)

Reply:
i've seen couple of places where logs get pushed to journal logs also.